src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,34 @@
+import { Person } from '../types';
+import PersonLink from './PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+const PeopleTable: React.FC<Props> = ({ people }) => {
+  const showPeople = people?.map(person => {
+    return <PersonLink person={person} people={people} key={person.slug} />;
```

Comment:
Consider using a more descriptive name than 'showPeople' for the variable holding the JSX elements. Something like 'personLinks' would more accurately describe its contents.